angular/Tooltip/Getting-Started.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Getting started for Angular Tooltip
-description: How to create the Tooltip in Angular
+title: Getting started with Angular Tooltip Control | Syncfusion
+description: Learn here about getting started with Angular Tooltip in Syncfusion Essential Angular Tooltip Control, its elements, and more.
```

Comment:
Learn here about getting started with Syncfusion Essential Angular Tooltip Control, its elements, and more.

Reply:
Done